zerver/webhooks/teamcity/view.py

```diff
@@ -39,7 +39,10 @@ def get_teamcity_property_value(property_list: List[Dict[str, str]], name: str)
 @has_request_variables
 def api_teamcity_webhook(request: HttpRequest, user_profile: UserProfile,
                          payload: Dict[str, Any]=REQ(argument_type='body')) -> HttpResponse:
-    message = payload['build']
+    message = payload.get('build')
+    if message is None:
+        # Ignore third-party specific (e.g. Slack/HipChat) payload formats
+        return json_success()
```

Comment:
Should this ideally instead call a "notify bot owner" function, like we've been adding for other use cases?  Like, this case means it's misconfigured, right?